src/client/components/equalizer/streamButton/streamButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StartStopStreamButton = (props) => {
```

Comment:
no need to create a component that, for eg defers from playButton component only with text